README.md

```diff
@@ -0,0 +1,47 @@
+DeDuplicated Queue MySQL Plugin
+===============================
+
+[![Build Status][travis-image]][Travis CI]
+[![Dependencies][dependencies-image]][Dependencies]
+[![Dev Dependencies][devdependencies-image]][Dev Dependencies]
+[![codecov.io][codecov-image]][Code Coverage]
+
+About
+-----
+
+A MySQL plugin for the DeDuplicated Queue (DDQ) module. The DDQ module and a whole host of documentation (including installation instructions) can be found here: https://github.com/tests-always-included/ddq
```

Comment:
I would make DeDuplicated Queue (DDQ) have a link pointing to the NPM page.

The last link should point to the NPM page as well. This will show the instructions and such as well as point them to the github page if they want to see the code.